tests/unittest/llmapi/test_llm_args.py

```
@@ -666,15 +666,15 @@ def test_extended_runtime_perf_knob_config_arbitrary_args(self):
     def test_cache_transceiver_config_arbitrary_args(self):
         """Test that CacheTransceiverConfig rejects arbitrary arguments."""
         # Valid arguments should work
-        config = CacheTransceiverConfig(backend="ucx",
+        config = CacheTransceiverConfig(backend="UCX",
```

Ruff (0.12.2)

`CacheTransceiverConfig` may be undefined, or defined from star imports

(F405)